### PR TITLE
RangeAdd: refactor to use useMutation hook

### PR DESCRIPTION
**Pull request by tbergq**

### CODE REVIEW

File: src/example-relay/src/mutations/RangeAdd/LocationsForm.js [hunk 1]
```diff
@@ -4,10 +4,11 @@
 
 import * as React from 'react';
 import { InputField, Stack, Button, Select } from '@kiwicom/orbit-components';
-import { useRelayEnvironment } from '@adeira/relay';
+import { graphql } from '@adeira/relay';
+// eslint-disable-next-line monorepo/no-internal-import
+import { useMutation } from '@adeira/relay/src/hooks';
```
Comment (mrtnzlml) [inline src/example-relay/src/mutations/RangeAdd/LocationsForm.js]: Can you export it, please? This looks like a mistake.

Comment (tbergq) [reply src/example-relay/src/mutations/RangeAdd/LocationsForm.js]: Will do

Comment (tbergq) [reply src/example-relay/src/mutations/RangeAdd/LocationsForm.js]: @mrtnzlml, looks ok now?

File: src/example-relay/src/mutations/RangeAdd/LocationsForm.js [hunk 2]
```diff
@@ -48,31 +61,51 @@ const getLocation = (state: State) => {
     type,
   };
 };
+
 export default (function LocationsForm() {
   const [state, dispatch] = React.useReducer(reducer, initialState);
-  const [loading, setLoading] = React.useState(false);
-  const env = useRelayEnvironment();
+  const [addLocation, loading] = useMutation(graphql`
```
Comment (mrtnzlml) [inline src/example-relay/src/mutations/RangeAdd/LocationsForm.js]: Tip: 

```patch
Index: src/example-relay/src/mutations/RangeAdd/LocationsForm.js
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- src/example-relay/src/mutations/RangeAdd/LocationsForm.js	(revision 511f28de876e4841b6889a1512e2cb13b4ff9025)
+++ src/example-relay/src/mutations/RangeAdd/LocationsForm.js	(date 1582622951172)
@@ -6,7 +6,7 @@
 import { InputField, Stack, Button, Select } from '@kiwicom/orbit-components';
 import { graphql, useMutation } from '@adeira/relay';
 
-import type { LocationsFormMutationResponse } from './__generated__/LocationsFormMutation.graphql';
+import type { LocationsFormMutation } from './__generated__/LocationsFormMutation.graphql';
 
 type Props = {||};
 
@@ -64,7 +64,7 @@
 
 export default (function LocationsForm() {
   const [state, dispatch] = React.useReducer(reducer, initialState);
-  const [addLocation, loading] = useMutation(graphql`
+  const [addLocation, loading] = useMutation<LocationsFormMutation>(graphql`
     mutation LocationsFormMutation($location: AddLocationInput!) {
       addLocation(location: $location) {
         __typename
@@ -93,7 +93,7 @@
     addLocation({
       configs,
       variables: { location: getLocation(state) },
-      onCompleted: (res: LocationsFormMutationResponse, errors: ?$ReadOnlyArray<Error>) => {
+      onCompleted: (res, errors) => {
         if (errors != null) {
           alert(errors.map(e => e.message).join(','));
         } else if (
```

**Pull request merged**